src/test/exit-codes/compute-phase-errors.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { toNano } from "@ton/core";
+import { Blockchain, SandboxContract, TreasuryContract } from "@ton/sandbox";
+import {
+    ComputePhaseErrorsTester as TestContract,
+    ExitCode4,
+} from "./contracts/output/compute-phase-errors_ComputePhaseErrorsTester";
+import "@ton/test-utils";
+
+describe("compute phase errors", () => {
+    let blockchain: Blockchain;
+    let treasure: SandboxContract<TreasuryContract>;
+    let contract: SandboxContract<TestContract>;
+
+    beforeEach(async () => {
+        blockchain = await Blockchain.create();
+        blockchain.verbosity.print = false;
+        treasure = await blockchain.treasury("treasure", {
+            resetBalanceIfZero: true,
+        });
+
+        contract = blockchain.openContract(await TestContract.fromInit());
+
+        const deployResult = await contract.send(
+            treasure.getSender(),
+            { value: toNano("10000") },
+            null,
+        );
+
+        expect(deployResult.transactions).toHaveTransaction({
+            from: treasure.address,
+            to: contract.address,
+            success: true,
+            deploy: true,
+        });
+    });
+
+    // 0: success
+    it("should test exit code 0", async () => {
```

Comment:
btw, you can even move the `it()` code to the helper

Reply:
Sure

Reply:
@anton-trunov

Reply:
Let's leave it as it is now then